### DISCUSSION TITLE
Live Preview remove Pop out functionality

### DISCUSSION THREAD

**Discussion by JacobViertel** [statamic/cms#7586] [Q&A]
Hi there,
I am using the custom rendering functionality of the Live Preview with Statamic 3.3.63 and Laravel Framework 8.83.27. Is it possible to remove the functionality 'Pop out' and Responsive somehow from the toolbar?

**Comment by jasonvarga** [comment]
No, it's not possible to remove it at the moment.

Why do you want to remove it?

**Reply by JacobViertel** [reply]
Thanks for the answer. I want to remove the button, as the functionality is not given. The live preview generates the correct html file. When clicking on the 'pop out' button a new window opens with debug outputs despite the html view. Afterwards, when I try to pop the window in again, nothing works anymore.
Experience-flow of the Screenshot:

1. I click on live preview and the live preview html is rendered correctly 
2. I click on Pop out and a second window opens up
3. The new window contains debug output and no longer the previousely rendered html. The main window also does no longer contain the html but the control panel view of my entry
4. When i click pop in, nothing happens. The entire page is broken and the buttons do no longer trigger any functionality

![Bildschirm­foto 2023-02-23 um 11 40 18](https://user-images.githubusercontent.com/56350358/220884105-6b99158f-9871-42e9-8b35-28850ab0e7ec.png)

**Reply by jasonvarga** [reply]
Can you record a video of it?

**Reply by JacobViertel** [reply]
Here is the video. I hope the quality is sufficient.
https://user-images.githubusercontent.com/56350358/221149854-1e7657eb-0d49-4e0b-85ee-912975792b63.mov
